scripts/analysis_0/nbconverted/3_visualize_data.py

```diff
@@ -266,13 +282,13 @@
 
 # ## Visualize multiple experiments in UMAP space
 
-# In[14]:
+# In[15]:
```

Comment:
It's a heart! lol

Reply:
yes!

As a note:  I remember we discussed why we didn't see the structure of the original data that is observed in UMAP in the PCA plots and why we are using PCA to visualize the "original data + technical noise".  So this is because PCA allows us to capture the technical variance that we have added to the data, which complements the line plot visualization that we have in our 2_ NBs.  The reason there is a difference between the two plots is just because we are only looking at the top 2 PCs and as opposed to UMAP which is attempting to capture the overall structure in the full dimensionality.

Reply:
👍